modules/ROOT/pages/dashboards/working-with-dashboards/index.adoc

```diff
@@ -0,0 +1,8 @@
+= Working with dashboards
```

Comment:
The entirety of the 'Working with Dashboards' section is out of scope for the internal preview, except for 'Managing dashboards'.